src/System.Device.Gpio/System/Device/Gpio/GpioController.cs

```diff
@@ -56,6 +56,11 @@ private int GetLogicalPinNumber(int pinNumber)
             return (NumberingScheme == PinNumberingScheme.Logical) ? pinNumber : _driver.ConvertPinNumberToLogicalNumberingScheme(pinNumber);
         }
 
+		public void SetDefaultGpioChip(int devGpioChip)
```

Comment:
>SetDefaultGpioChip [](start = 14, length = 18)

two things here: 

1- should we allow passing the default gpio chip to the controller constructor? even as a default parameter?

2- should we convert this method to a property with setter and getter? something like 


public int DefaultGpioChip { set; get; }

Reply:
Thanks for the review @tarekgh 
I liked the approach of passing that in the constructor of LibGpiodController

Reply:
1 was implemented on 940bc17